CHANGES/8270.bugfix.rst

```diff
@@ -0,0 +1,9 @@
+Fix ``AsyncResolver`` to match ``ThreadedResolver`` behavior
+-- by :user:`bdraco`.
+
+On system with IPv6 support, the :py:class:`~aiohttp.resolver.AsyncResolver` would not fallback
```

Comment:
Is it linkable? Looking at https://webknjaz.github.io/intersphinx-untangled/docs.aiohttp.org/, it doesn't appear so. You'd need to add it to https://github.com/aio-libs/aiohttp/blob/1542348/docs/conf.py#L396 or document in RST. Otherwise, I expect Sphinx to choke: https://github.com/aio-libs/aiohttp/actions/runs/8499653747/job/23280716056?pr=8270#step:13:129.

Reply:
Thanks. Sadly it looks like they are undocumented so 3d5175f2ddeec003d01793a4c7bd7319c7abc25d

Reply:
Yeah, that's why I initially suggested double backticks as this is the easiest way to highlight the identifiers without having to deal with Sphinx. Marking as resolved.